portfolio/src/main/webapp/index.html

```diff
@@ -79,50 +79,66 @@ <h3>UCSD Triton</h3>
             <div class="row">
                 <div class="column">
                     <div class="content">
-                        <img src="/images/step.jpg" alt="Google STEP Logo" style="width:100%">
+                        <img src="/images/step.jpg" alt="Google STEP Logo" border=1 style="width:100%">
```

Comment:
I think it will be cleaner to create a class for img content and move the styling to there

Reply:
I created a new CSS command for `content > img` objects that creates the border and removed the border component on img tags in the HTML